uniflow/flow/client.py

```diff
@@ -0,0 +1,73 @@
+"""All Uniflow Clients"""
+
+from dataclasses import asdict
+from typing import Any, List, Mapping
+
+from uniflow.flow.config import ExtractConfig, TransformConfig
+from uniflow.flow.server import ExtractServer, TransformServer
+
+
+class ExtractClient:
+    """Uniflow Extract Client"""
+
+    def __init__(self, config: ExtractConfig) -> None:
+        """Client constructor
+
+        Args:
+            config (Config): Config for the flow
+
+        """
+        self._config = config
+        self._server = ExtractServer(asdict(self._config))
+
+    def run(self, input_list: List[Mapping[str, Any]]) -> List[Mapping[str, Any]]:
+        """
+        Run the flow
+
+        Args:
+            input_list (List[Mapping[str, Any]]): List of inputs to the flow
+
+        Returns:
+            List[Mapping[str, Any]]: List of outputs from the flow
+        """
+        # convert config to dict for future remote calls
+        output = self._server.run(input_list)
+        return output
+
+    def async_run(self) -> None:
+        """Run the flow asynchronously"""
+        # TODO: Implement async client
+        print("Client running async")
+
+
+class TransformClient:
+    """Uniflow Client"""
+
+    def __init__(self, config: TransformConfig) -> None:
+        """Client constructor
+
+        Args:
+            config (Config): Config for the flow
+
+        """
+        self._config = config
+        self._server = TransformServer(asdict(self._config))
+
+    def run(self, input_list: List[Mapping[str, Any]]) -> List[Mapping[str, Any]]:
+        """
+        Run the flow
+
+        Args:
+            input_list (List[Mapping[str, Any]]): List of inputs to the flow
+
+        Returns:
+            List[Mapping[str, Any]]: List of outputs from the flow
+        """
+        # convert config to dict for future remote calls
+        output = self._server.run(input_list)
+        return output
+
+    def async_run(self) -> None:
+        """Run the flow asynchronously"""
+        # TODO: Implement async client
+        print("Client running async")
```

Comment:
nit: setup you vs code properly by following this [notion](https://www.notion.so/goldpiggy/Python-Linter-Setup-30fb3b81f0904af889832e4c697c5ec9?pvs=4) to avoid this linting error such as need a new line.